Essentials/src/net/ess3/api/events/TPARequestEvent.java

```diff
@@ -0,0 +1,56 @@
+package net.ess3.api.events;
+
+
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+import com.earth2me.essentials.CommandSource;
+
+import net.ess3.api.IUser;
+
+
+public class TPARequestEvent extends Event implements Cancellable {
```

Comment:
@drtshock @SupaHam Thoughts on having a `showCancellationMessage` boolean defaulting to `true` that determines whether EssentialsX shows its own "request cancelled" message, in case a plugin wants to display its own message instead?